feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/followeeMoodEvent.java

```diff
@@ -0,0 +1,29 @@
+package com.example.feelslikemonday.model;
+
+/**
+ * This is a model class for every user that your follow
+ * It keeps track of their username and most recent Mood event
+ */
+public class followeeMoodEvent {
```

Comment:
We should not need this class, unless if there is a better reason as to, my understanding is that we go (UserPref to get user's username, use username to get users usernames that we are following, use those usernames to get those other users, and then get the most recent mood from those users)

Reply:
This class makes it easier to keep track of all the people you follow because you're not really following an entire user, you are only following their most recent mood event. I agree that you can probably do it without this class but I still feel that having a list of objects that keep track of each mood event you follow along with the username is a cleaner and easier way to keep track of your followee's

Reply:
Yeah, fair point, it would be a bit complicated working with just Users